jubilant/_juju.py

```diff
@@ -237,6 +237,18 @@ def add_secret(
 
         return SecretURI(output.strip())
 
+    def add_ssh_key(self, *keys: str) -> None:
+        """Add SSH keys to the model.
+
+        The SSH keys are added to all current and future machines in the model.
+
+        Args:
+            keys: SSH public key or keys to add. Each key should be the full
+                SSH public key string (for example, ``ssh-rsa AAAAB3... user@host``).
+        """
+        args = ['add-ssh-key', *keys]
+        self.cli(*args)
```

Comment:
Simplification, as we do in some of the simpler commands:
```suggestion
        self.cli('add-ssh-key', *keys)
```